### ISSUE TITLE
entrapment

### ISSUE THREAD

**Issue by FellowTraveler** [liberland/constitution#67]
Should have a provision banning entrapment by law enforcement. For example, agents of the law should not be able to enlist people to commit crimes and then charge them with those crimes. Often in the United States, agents will hire people to transport drugs and then charge them with transporting drugs. Etc.

If you are amenable to this, I can write the appropriate pull request.

**Comment by KacperZajc** [comment]
I think this is a good idea, we should think about that.

**Comment by KacperZajc** [comment]
No person enlisted by any agent of the Public Administration to commit any criminal offence shall be tried for that offence.

???

**Comment by FellowTraveler** [comment]
That sounds great to me!

**Comment by KacperZajc** [comment]
anyone else wants to contribute?

**Comment by jasoneriksen** [comment]
I concur that the abuses by government agencies (typically the DEA) here in the United States in this regard are appalling. 

But I can see situations where the enlisted party could use this entrapment clause as a legal protection. How does this apply in situations where a corrupt police officer enlists a gang to burn down a business that hasn't paid its protection fee for the week?

**Comment by Zephram-C** [comment]
I am not sure this is a good idea. This way enlisted crimes may be committed but never punished (and victims restituted by law). If I want to commit crime, all I need for impunity is enlistment by colluding agent. What if one agent of the Public Administration enlists another one to commit crime? 

*agents of the law should not be able to enlist people to commit crimes*
This is simple enough and it is already included in **Provisional Administration of Justice - Part II - 7**
*A person who is procuring, assisting or attempting to commit a criminal offence is to be charged as if he committed the particular crime. The punishment must be adequate according to his contribution to the crime.*

**Comment by KacperZajc** [comment]
so:

No agent of the Public Administration shall enlist any person to commit any criminal offence.

is that better?

**Comment by Zephram-C** [comment]
I think that abuse (eg. by DEA) happens because Public Agents have immunity in such cases. (Clearly any non-Public Agent enlisting someone to transport drugs would be punished [in drug intolerant non-Liberland].) Take away that immunity and we do not need special provision.

**Comment by FellowTraveler** [comment]
I agree with @jasoneriksen that there are situations where the enlisted party could use this entrapment clause as a legal protection. He has a very good point.

I agree with @KacperZajc to therefore simplify the wording to *"No agent of the Public Administration shall enlist any person to commit any criminal offence."*

I agree with @Zephram-C that public agents should not have immunity for crimes they enlist others to commit. He mentions "Provisional Administration of Justice - Part II - 7". Is there any reason not to add that wording directly into the Constitution itself?

**Comment by FellowTraveler** [comment]
I just want to make sure that whatever solution we come up with, would prevent this sort of thing from happening:  http://www.theguardian.com/us-news/2015/may/12/polygraph-critic-trial-lie-busters-doug-williams

**Comment by FellowTraveler** [comment]
Look what else these assholes did, in the same case as above: "After Williams’s business was raided, CBP shared his business’s database of nearly 5,000 customers, including their social security numbers and addresses, with 30 other federal agencies."
